source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -64,11 +71,28 @@ def __contains__(self, obj):
 
 class Document(ia2Web.Document):
 
-	def _get_treeInterceptorClass(self):
-		states = self.states
-		if controlTypes.State.EDITABLE not in states and controlTypes.State.BUSY not in states:
-			return ChromeVBuf
-		return super(Document, self).treeInterceptorClass
+	def _get_treeInterceptorClass(self) -> typing.Optional["TreeInterceptor"]:
+		shouldLoadVbufOnBusyFeatureFlag = bool(
+			config.conf["virtualBuffers"]["loadChromiumVbufOnBusyState"]
+		)
+		vbufUnavailableStates = {  # if any of these are in states, don't return ChromeVbuf
+			controlTypes.State.EDITABLE,
+		}
+		if not shouldLoadVbufOnBusyFeatureFlag:
+			log.debug(

```

Comment:
No, only once per doc load. If we find that they are noisy, then we can revisit. Initially I think it will be helpful for debugging if it is very clear.